packages/sonarwhal/src/lib/cli/rules/templates/new-rule/tsconfig.json.hbs

```diff
@@ -0,0 +1,40 @@
+{{#if official}}{
+    "compilerOptions": {
+        "outDir": "dist"
+    },
+    "exclude": [
+        "dist",
+        "node_modules"
+    ],
+    "extends": "../../tsconfig.json",
+    "include": [
+        "src/**/*.ts",
+        "tests/**/*.ts"
+    ]
+}{{else}}{
+    "compilerOptions": {
```

Comment:
I can't use the one in the root of the project because if it is an external rule the file will not exist in the published package 😞